gobuffalo/azuredeploy.json

```diff
@@ -0,0 +1,196 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "imageName" : {
+        "type": "String",
+        "defaultValue": "appsvc/sample-hello-world:latest",
+        "metadata": {
+          "description": "The image which defines the application to be hosted in Azure."
+        }
+      },
+      "name": {
+        "type": "String",
+        "defaultValue": "[concat('site', uniqueString(resourceGroup().id, deployment().name))]",
+        "metadata": {
+          "description": "A unique name to identify the site, and its relate assets once it is created."
+        }
+      },
+      "database": {
+        "type": "String",
+        "defaultValue": "none",

```

Comment:
use a default that includes a db, so we can cover that path

Reply:
Fixed locally.